README.md

```diff
@@ -42,11 +42,12 @@ Create a new git tag with the new version and [create a release based on the tag
 ## Available hooks
 
 - `black`: Python Code Formatter
+- `flake8`: Tool For Python Style Guide Enforcement
+- `isort`: Python import sorting
 - `circleci-config-validate`: Test if the CircleCI config file is well formed.
 - `circleci-config-pack`: Pack the CircleCI config and build the config.yml
 - `detect-secrets`: Scan for secrets committed into the repo
 - `end-of-file-fixer`: Checks that all your JSON files are pretty.
-- `flake8`: Tool For Style Guide Enforcement
 - `google-java-code-format`
 - `poetry-pytest`: Use poetry to create a virtual-env and run pytest
 - `pretty-format-json`: Checks that all your JSON files are pretty.
```

Comment:
They were in alphabetical order!

Reply:
I didnt notice xD I put all python things together

Reply:
poetry-pytest is still sitting alone 😄